rpcs3/rpcs3qt/log_frame.cpp

```diff
@@ -226,6 +226,15 @@ void log_frame::CreateAndConnectActions()
 
 	m_clearTTYAct = new QAction(tr("Clear"), this);
 	connect(m_clearTTYAct, &QAction::triggered, m_tty, &QTextEdit::clear);
+	
+	m_stackAct_tty = new QAction(tr("Stack Mode (TTY)"), this);
+	m_stackAct_tty->setCheckable(true);
+	connect(m_stackAct_tty, &QAction::toggled, xgui_settings.get(), [=](bool checked)
+	{
+		xgui_settings->SetValue(gui::l_stack_tty, checked);
+		m_stack_tty = checked;
+	});
+
```

Comment:
Emtpy line

Reply:
I can remove the empty line there, but it's formatted with an empty line at the block I copied from at 279